docs/topics/ksp/ksp-quickstart.md

```diff
@@ -99,75 +192,62 @@ dependencies {
 }
 ```
 
-### Setup using Groovy
+</tab>
+<tab title="Groovy" group-key="groovy">
 
-1. Create another module that contains a workload where you want to try out your processor.
-
-  ```groovy
-  pluginManagement {
-    repositories {
-        gradlePluginPortal()
-    }
-  }
-  ```
-
-2. In your projects `build.gradle` file add a plugins block containing the ksp plugin:
-
-  ```groovy
-  plugins {
-    id "com.google.devtools.ksp" version "1.5.30-1.0.0-beta09"
-  }
-  ```
-  
-3. In the modules `build.gradle`, add the following:
-
-    * Apply the `com.google.devtools.ksp` plugin:
+```groovy
+plugins {
+    id 'com.google.devtools.ksp' version '%kspSupportedKotlinVersion%-%kspVersion%'
+    id 'org.jetbrains.kotlin.jvm' version '%kotlinVersion%'
+}
 
-    ```groovy
-    apply plugin: 'com.google.devtools.ksp'
-    ```
+repositories {
+    mavenCentral()
+}
 
-    * Add `ksp <your processor>` to the list of dependencies.
+dependencies {
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib:%kotlinVersion%'
+    implementation project(':test-processor')
+    ksp project(':test-processor')
+}
+```
 
-    ```groovy
-    dependencies {
-        implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-        implementation project(":test-processor")
-        ksp project(":test-processor")
-    }
-    ```
+</tab>
+</tabs>
 
 ## Pass Options to Processors
```

Comment:
Should this be capitalized? I mean "Options" and "Processors"

Reply:
Thank you, missed that. [Fixed.](https://github.com/JetBrains/kotlin-web-site/pull/2570/commits/e41975ede9c199e8c5e4d1767b422037bb9675e9)